Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM node:18-alpine AS production
+
+#RUN apt-get update
+#RUN apt-get upgrade -y
+
+ARG NYPL_HEADER_URL
+
+WORKDIR /app
+
+
+# Set environment variables. NODE_ENV is set early because we
+# want to use it when running `npm install` and `npm run build`.
+ENV PATH /app/node_modules/.bin:$PATH 
+# \
+#     NODE_ENV=production
+
+# Set the timezone
+# ENV TZ=America/New_York
+# RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
I think this was done in a project because of timezones affecting unit tests. Do you think DC will have timezone issues? If not, we can remove it.